src/common/dependency-manager/local-manager.ts

```diff
@@ -118,13 +120,15 @@ export default class LocalDependencyManager extends DependencyManager {
     const dependency_resolvers = [];
     for (const [dep_name, dep_id] of Object.entries(parent_node.service_config.getDependencies())) {
 
-      const env_services = this.environment.getServiceDetails(`${dep_name}:${dep_id}`);
-      if (env_services?.host && env_services?.port) {
-        await this.loadExternalService(env_services, `${dep_name}:${dep_id}`);
+      const env_service = this.environment.getServiceDetails(`${dep_name}:${dep_id}`);
+      if (env_service!.getInterfaces()) {
```

Comment:
this looks like a real lint issue. should be `if (env_service?.getInterfaces()) {`

take a look at the github actions lint warnings